src/Application.php

```diff
@@ -677,24 +677,16 @@ private function checkForDuplicateRoute($path, $methods = null)
      */
     private function swallowDeprecationNotices()
     {
-        $handler = function ($errno, $errstr) {
-            if ($errno !== E_USER_DEPRECATED) {
-                return false;
+        $handler = function ($errno, $errstr, $errfile, $errline, $errcontext) use (&$previous) {
```

Comment:
You'll need to define `$previous` prior to declaring this handler; references can sometimes lead to issues otherwise.